### PR TITLE
#3268 - Local connection to the openshift DB - Makefile command change

### PR DESCRIPTION
**Pull request by guru-aot**
- [x] Local connection to the openshift DB - Makefile command change
- [x] Commented uninstall commands in the makefile of clamav and crunchy for security.

### CODE REVIEW

File: sources/Makefile [hunk 1]
```diff
@@ -243,21 +243,33 @@ workers:
...
-	@oc port-forward services/patroni-master 5656:5432
+	@oc port-forward services/simsdb-pgbouncer 5656:5432
+
+connect-stg-db:
+	@echo "Forward local port 5757 to Openshift simsdb-pgbouncer default port."
+	@echo "Use your prefered DB Manager (e.g. pgAdmin or DBeaver) to connect to the Postgres database using the local port 5757."
+	@oc project a6ef19-test
+	@oc port-forward services/simsdb-pgbouncer 5757:5432
+
+connect-prod-db:
```
Comment (sh16011993) [inline sources/Makefile:271]: Thank you for adding these 👍

File: sources/Makefile [hunk 2]
```diff
@@ -243,21 +243,33 @@ workers:
...
 
 connect-qa-db:
-	@echo "Forward local port 5656 to Openshift Patroni-master default port."
+	@echo "Forward local port 5656 to Openshift simsdb-pgbouncer default port."
 	@echo "Use your prefered DB Manager (e.g. pgAdmin or DBeaver) to connect to the Postgres database using the local port 5656."
 	@oc project 0c27fb-test
-	@oc port-forward services/patroni-master 5656:5432
+	@oc port-forward services/simsdb-pgbouncer 5656:5432
+
+connect-stg-db:
```
Comment (sh16011993) [inline sources/Makefile:265]: Thank you for adding these 👍

**Review by sh16011993 — APPROVED** [review]
LGTM @guru-aot 👍 Thank You for adding the connection script for staging and prod.

File: sources/Makefile [hunk 3]
```diff
@@ -243,21 +243,33 @@ workers:
...
-	@oc port-forward services/patroni-master 5555:5432
+	@oc port-forward services/simsdb-pgbouncer 5555:5432
 
 connect-qa-db:
-	@echo "Forward local port 5656 to Openshift Patroni-master default port."
+	@echo "Forward local port 5656 to Openshift simsdb-pgbouncer default port."
 	@echo "Use your prefered DB Manager (e.g. pgAdmin or DBeaver) to connect to the Postgres database using the local port 5656."
 	@oc project 0c27fb-test
-	@oc port-forward services/patroni-master 5656:5432
+	@oc port-forward services/simsdb-pgbouncer 5656:5432
```
Comment (andrewsignori-aot) [inline sources/Makefile:263]: We no longer have QA DB on `0c27fb-test`. We should either remove `connect-qa-db` or change the `@oc project 0c27fb-test`.

File: devops/helm/clam-av/main/Makefile [hunk 4]
```diff
@@ -36,10 +36,10 @@ lint: helm-dep
...
 	@helm upgrade --dry-run --install $(call arguments)
 
-.PHONY: uninstall
-uninstall: helm-dep
-uninstall:
-	@helm uninstall ${NAME} -n ${NAMESPACE}
+#.PHONY: uninstall
+#uninstall: helm-dep
+#uninstall:
+#	@helm uninstall ${NAME} -n ${NAMESPACE}
```
Comment (andrewsignori-aot) [inline devops/helm/clam-av/main/Makefile:42]: What is the possible security risk to keep those? Is it about someone using the GitHub actions and selecting the "Unistall" option or executing the command from local machine?

Comment (andrewsignori-aot) [reply devops/helm/clam-av/main/Makefile:42]: If we leave a comment code in the source code it must have a nice comment explaining why and when it should be uncommented.

**Review by andrewsignori-aot — COMMENTED** [review]
Nice changes, please take a look at the comments.
Do we need an alternative command for a super user connection that will fail using `pgbouncer`?

File: devops/helm/clam-av/main/Makefile (hunk 4 above)
Comment (guru-aot) [reply devops/helm/clam-av/main/Makefile:42]: From local machine

File: sources/Makefile (hunk 3 above)
Comment (guru-aot) [reply sources/Makefile:263]: Yes i missed it, i believe we may need to remove the connect-qa-db

**Comment by guru-aot** [comment]
> Nice changes, please take a look at the comments. Do we need an alternative command for a super user connection that will fail using `pgbouncer`?

Sure, the connection to the super user connection will he given in the wiki documentation. Also most of the operations should be available to the non-super-user user we have in the DB, as its ownership is given to all the objects and the db migration is run in them.

File: sources/Makefile [hunk 5]
```diff
@@ -243,21 +243,27 @@ workers:
...
-connect-qa-db:
-	@echo "Forward local port 5656 to Openshift Patroni-master default port."
-	@echo "Use your prefered DB Manager (e.g. pgAdmin or DBeaver) to connect to the Postgres database using the local port 5656."
-	@oc project 0c27fb-test
-	@oc port-forward services/patroni-master 5656:5432
+connect-stg-db:
+	@echo "Forward local port 5757 to Openshift simsdb-pgbouncer default port."
+	@echo "Use your prefered DB Manager (e.g. pgAdmin or DBeaver) to connect to the Postgres database using the local port 5757."
+	@oc project a6ef19-test
+	@oc port-forward services/simsdb-pgbouncer 5757:5432
```
Comment (andrewsignori-aot) [inline sources/Makefile:263]: Since QA was removed, please keep the ports sequence.

**Review by andrewsignori-aot — APPROVED** [review]
Thanks for making the changes, looks good 👍

**Pull request merged**